ToDoWebApi/Applications/UserOperations/Queries/GetUserDetail/GetUserDetailQuery.cs

```diff
@@ -0,0 +1,48 @@
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using ToDoWebApi.DbOperations;
+
+namespace ToDoWebApi.Applications.UserOperations.Queries.GetUserDetail
+{
+    public class GetUserDetailQuery
+    {
+        public const string ExceptionMessage = "User does not exist.";
+        
+        public int UserId { get; set; }
```

Comment:
UserId değil Id yapalım property'nin ismini, sadece User için kullanılan bir class sonuçta.

Reply:
Değiştirildi.